Makefile

```
@@ -9,4 +11,18 @@ install/observability-core:
 
 # Create the whitelist for model metrics
 install/observability-edge:
-  kustomize build acm/odh-edge/acm-observability | kubectl apply -f
\ No newline at end of file
+	kustomize build acm/odh-edge/acm-observability | kubectl apply -f
+
+# Test all kustomize files build sucessfully
+test/kustomize:
+	@for dir in `find . -name kustomization.yaml -exec dirname {} \;`; do \
+		echo "Testing $$dir"; \
+		cd $$dir; \
+		kustomize build . > /dev/null; \
```

```suggestion
		oc apply --dry-run=server -k . > /dev/null; \
```

^Assuming `oc` has access to a cluster to "dry-run" apply, this will error on the naming issue I mentioned in another comment in model-deployment.yaml